.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,23 @@
+name: Update Breaking Change Status
+on:
+  pull_request:
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.id == '4598495472'
```

Comment:
Can you add a comment indicating what this magic number corresponds to?

Reply:
done!